src/Init/Data/Nat/Lemmas.lean

```diff
@@ -916,6 +919,8 @@ theorem mul_add_mod (m x y : Nat) : (m * x + y) % m = y % m := by
   · exact (m % 0).div_zero
   · case succ n => exact Nat.div_eq_of_lt (m.mod_lt n.succ_pos)
 
+theorem shiftLeft_eq_mul_pow (m) : ∀ n, m <<< n = m * 2 ^ n := shiftLeft_eq _
```

Comment:
If you can prove this theorem just as an alias to `shiftLeft_eq` why is it restated under a new name here?